src/ethereum_test_rpc/rpc.py

```diff
@@ -402,10 +460,13 @@ def get_blobs(
         version: int,
     ) -> GetBlobsResponse:
         """`engine_getBlobsVX`: Retrieves blobs from an execution layers tx pool."""
+        method = f"getBlobsV{version}"
+        params = [f"{h}" for h in versioned_hashes]
+
         return GetBlobsResponse.model_validate(
             self.post_request(
-                f"getBlobsV{version}",
-                [f"{h}" for h in versioned_hashes],
+                method=method,
+                params=params,
```

Comment:
```suggestion
                params=[params],
```
The parameter is a list of lists, `tests/osaka/eip7594_peerdas/test_get_blobs.py`  stopped working because of this.